js/tools.js

```diff
@@ -706,6 +706,18 @@ function getStorageAvailable() {
 	}
 }
 
+/**
+ * Refresh bulk actions combination number after creating or deleting combinations
+ *
+ * @param sign - int
+ * @param number - int
+ */
+var refreshTotalCombinations = function (sign, number) {
+  var currentnumber;
+  currentnumber = parseInt($('#js-bulk-combinations-total').text()) + (sign * number);
```

Comment:
This line can be merged with the previous one

Reply:
The selector `$('#js-bulk-combinations-total')` is performed twice. Please store it in a variable.